lib/main/atom/atomConfig.ts

```diff
@@ -0,0 +1,31 @@
+import {getName} from "../lang/utils";
+
+interface IConfig extends AtomCore.IConfig {
+  set<T>(keyPath:string, value: T):any;
+}
+// Documentation https://atom.io/docs/api/v0.177.0/Config and http://json-schema.org/examples.html
+// To add a new setting you need to add to
+//    schema
+//    getter/setter
+
+var packageName = 'atom-typescript';
+function getConfig<T>(nameLambda: () => any): T {
+    return atom.config.get(packageName + '.' + getName(nameLambda));
+}
+function setConfig<T>(nameLambda: () => any, value: T): T {
+    return (atom.config as IConfig).set(packageName + '.' + getName(nameLambda), value);
+}
+
+class Config {
+    schema = {
+        showSemanticView: {
```

Comment:
first I did remove the config, but now is re-added (in `lib/main/atomts.ts`):  
I see no other way, how we could remember the user's wish/setting, when Atom is started the next time.

I.e. the plugin should remember, if the user did have the Semantic View open (or not), the next time, Atom is started.